### DISCUSSION TITLE
Welcome to create-aio-app Discussions!

### DISCUSSION THREAD

**Discussion by webknjaz** [aio-libs/create-aio-app#214] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by piyushsatti** [comment]
Hi! I was trying to run the software using the contribution document provided in the repo however I have been facing an issue that might be trivial. I was looking for a place to make community posts or ask simpler doubts perhaps. Slack or otherwise, does that not exist yet or am I looking at the wrong places. And if not, where can I post the same if required.
